test/sed.js

```diff
@@ -170,7 +170,7 @@ test('glob file names, with in-place-replacement', t => {
   const result = shell.sed('-i', 'test', 'hello', `${t.context.tmp}/file*.txt`);
   t.falsy(shell.error());
   t.is(result.code, 0);
-  t.is(result.toString(), 'hello1\n\nhello2\n'); // TODO: fix sed's behavior
```

Comment:
I think this TODO is about the double newline between `hello1` and `hello2`, not the `-i` behavior. Could you file a github issue to track the double-newline thing?

Reply:
https://github.com/shelljs/shelljs/issues/1068